Integrations/IntezerV2/IntezerV2.py

```diff
@@ -0,0 +1,252 @@
+SERVER_URL = 'https://analyze.intezer.com/api'
+API_VERSION = 'v2-0'
+IS_AVAILABLE_URL = 'is-available'
+ERROR_PREFIX = 'Error from Intezer:'
+ACCEPTABLE_HTTP_CODES = {200, 201, 202}
+
+http_status_to_error_massage = {
+    400: '400 Bad Request - Wrong or invalid parameters',
+    401: '401 Unauthorized - Wrong or invalid api key',
+    403: '403 Forbidden - The account is not allowed to preform this task',
+    404: '404 Not Found - Analysis was not found',
+    410: '410 Gone - Analysis no longer exists in the service',
+    500: '500 Internal Server Error - Internal error',
+    503: '503 Service Unavailable'
+}
+dbot_score_by_verdict = {'malicious': 3,
+                         'suspicious': 2,
+                         'trusted': 1,
+                         'neutral': 1,
+                         'no_threats': 1}
+
+''' HELPER FUNCTIONS '''
+
+
+def handle_response(response, acceptable_http_status_codes):
+    if response.status_code not in acceptable_http_status_codes:
+        return_error('{} {}'.format(ERROR_PREFIX, http_status_to_error_massage.get(response.status_code,
+                                                                                   'Failed to perform request')))
+
+    return response.json()
+
+
+def get_session():
+    response = requests.post(SERVER_URL + '/v2-0/get-access-token', json={'api_key': API_KEY})
+    response = handle_response(response, {200})
+    session = requests.session()
+    session.headers['Authorization'] = 'Bearer {}'.format(response['result'])
+
+    return session
+
+
+''' COMMANDS '''
+
+
+def check_is_available():
+    session = get_session()
+    url = '{}/{}'.format(SERVER_URL, IS_AVAILABLE_URL)
+    result = session.get(url)
+
+    return 'ok' if result.json()['is_available'] else None
+
+
+def analyze_by_hash():
+    file_hash = demisto.getArg('file_hash')
+    response = make_analyze_by_hash_request(file_hash)
+    handle_analyze_by_hash_response(response, file_hash)
+
+
+def make_analyze_by_hash_request(file_hash):
+    session = get_session()
+    data = {'hash': file_hash}
+    return session.post(SERVER_URL + '/v2-0/analyze-by-hash', json=data)
+
+
+def handle_analyze_by_hash_response(response, file_hash):
+    if response.status_code == 404:
+        dbot = {
+            'Vendor': 'Intezer',
+            'Type': 'hash',
+            'Indicator': file_hash,
+            'Score': 0
+        }
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'EntryContext': {'DBotScore': [dbot]},
+            'HumanReadable': 'Hash {} does not exist on Intezer genome database'.format(file_hash),
+            'ContentsFormat': formats['json'],
+            'Contents': ''
+        })
+        return
+
+    elif response.status_code == 400:
+        return_error('File hash is not valid.\nIntezer file hash reputation supports only sha256, '
+                     'sha1 and md5 hash formats.\n')
+
+    handle_analyze_response(response)
```

Comment:
Not sure what you meant to do here.
In case the status code is 404, you return DBot score, and otherwise you don't.
Can you explain the logic please?